lib/utils/scope-util.js

```diff
@@ -0,0 +1,16 @@
+const isFormat = require('@node-oauth/formats');
+const InvalidScopeError = require('../errors/invalid-scope-error');
+
+module.exports = {
+  parseScope: function (requestedScope) {
+    if (!isFormat.nqschar(requestedScope)) {
+      throw new InvalidScopeError('Invalid parameter: `scope`');
+    }
+
+    if (requestedScope == null) {
```

Comment:
is this block ever reachable? I think `null` would fail the above test `isFormat.nqschar`, right?

Reply:
`null` is allowed in the above check